src/font/lv_font.c

```diff
@@ -66,18 +66,41 @@ bool lv_font_get_glyph_dsc(const lv_font_t * font_p, lv_font_glyph_dsc_t * dsc_o
 {
     LV_ASSERT_NULL(font_p);
     LV_ASSERT_NULL(dsc_out);
-    dsc_out->resolved_font = NULL;
+    const lv_font_t * placeholder_font = NULL;
     const lv_font_t * f = font_p;
-    bool found = false;
+
+    dsc_out->resolved_font = NULL;
+
     while(f) {
-        found = f->get_glyph_dsc(f, dsc_out, letter, letter_next);
-        if(found && !dsc_out->is_placeholder) {
-            dsc_out->resolved_font = f;
-            break;
+        bool found = f->get_glyph_dsc(f, dsc_out, letter, letter_next);
+        if(found) {
+            if(!dsc_out->is_placeholder) {
+                dsc_out->resolved_font = f;
+                return true;
+            }
+            else if(placeholder_font == NULL) {
+                placeholder_font = f;
+            }
         }
         f = f->fallback;
     }
-    return found;
+
+    if(placeholder_font != NULL) {
```

Comment:
I meant this. :slightly_smiling_face: 

A minor optimization issue is that now `get_glyph_dsc` is called twice on the placeholder font. So `dsc_out` can be saved too, but it doesn't matter much as probably there are only a few missing characters.

Reply:
A second call to `get_glyph_dsc()` is necessary for FreeType interface. Before calling `lv_font_get_glyph_bitmap()`, need make sure that the `get_glyph_dsc()` of the current glyph is called.  `font->fallback()`  may erase the bitmap corresponding to the previous graph. So we have to call `get_glyph_dsc()`  again at the end.

Reply:
I see, okay.